amf-webapi/shared/src/main/scala/amf/client/environment/ApiEnvironment.scala

```diff
@@ -0,0 +1,30 @@
+package amf.client.environment
+
+import amf.client.remod.AMFEnvironment
+import amf.plugins.document.graph.AMFGraphParsePlugin
+import amf.plugins.document.webapi._
+
+// defined private unit mayor version is released
+private[amf] object ApiEnvironment {
```

Comment:
Hmm suspicious. With this we are forcing adopters to depend on all of the artifacts we support. I agree that we have to make it easy for adopters to migrate but still, suspicious.